examples/contrib/oed/eig_estimation_benchmarking.py

```diff
@@ -233,7 +236,14 @@
              {"num_theta_samples": 60, "num_y_samples": 2, "num_steps": 1200, "final_num_samples": 100,
               "classifier": (LinearModelClassifier, {"scale_tril_init": 1/3., "ntheta": 60}),
               "optim": (optim.Adam, {"optim_args": {"lr": 0.05}})}),
-            (truth_lm, {})
+            (truth_lm, {}),
+            (laplace,
+             {"num_samples": 1,
```

Comment:
could this ever be different from `1`? This is for the MAP estimation, i.e. expectation with respect to a delta, so perhaps we should hard-code it to be `1`

Reply:
Good point, I've hard-coded it.